src/main/java/com/teamcautionrobotics/robot/Cargo.java

```diff
@@ -21,7 +23,10 @@ private CargoMoverSetting(double power) {
     // Attached to mechanum wheels in front of the robot that allow for more driver
     // error.
     private final Solenoid funnelRollerDeployer;
-    private final Solenoid exitFlapDeployer;
+    private Solenoid exitFlapDeployer;
+    private DoubleSolenoid doubleExitFlapDeployer;
+
+    private boolean usingDoubleSolenoids;
```

Comment:
This should be final, since its value does not change after the constructor finishes.